### PR TITLE
Bruce 86

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: OxfordBodleian/bruce/BDLbruce86.xml [hunk 1]
```diff
@@ -72,382 +76,384 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                           <locus from="165rc" to="168vc"/>
                            <title type="complete" ref="LIT1395Fethan#Appendix" xml:lang="gez">በእንተ፡
                               ተዋርሶ፡</title>
                            <textLang mainLang="gez"/>
                         </msItem>
 
-                        <colophon xml:id="coloph1" xml:lang="gez"> 
+                        <colophon xml:id="coloph1" xml:lang="gez">
+                           <locus from="168vc" to="160rb"/>
                            መልዓ፡ ጽሒፈ፡ ዝንቱ፡ መጽሐፈ፡ ቀኖና፡ <sic resp="PRS3562Dillmann">ዘፈትሐ፡</sic> ኵነኔ፡ ሕግ፡ መሲሐዊት፡ <date calendar="world">በዕለተ፡ ሠሉስ፡ አመ፡ ፲ወ፱ ለወርኃ፡ ኃምሌ፡ በ፸፻ወ፩፻፸ወ፰ ዓመተ፡
```
Comment (DariaElagina) [inline OxfordBodleian/bruce/BDLbruce86.xml]: to 169rb?

File: OxfordBodleian/bruce/BDLbruce86.xml [hunk 2]
```diff
@@ -503,255 +512,258 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                        <locus from="211va" to="212ra"/>
                         <title type="complete" ref="LIT4017Mashafa#Chapter29" xml:lang="gez">በእንተ፡
                            ሜሮን፡</title>
                         <textLang mainLang="gez"/>
                      </msItem>
 
                      <msItem xml:id="ms_i2.31">
-                        <locus target="#212"/>
+                        <locus from="212ra" to="121vc"/>
                         <title type="complete" ref="LIT4017Mashafa#Chapter30" xml:lang="gez">በእንተ፡
```
Comment (DariaElagina) [inline OxfordBodleian/bruce/BDLbruce86.xml]: to 212vc?

**Comment by thea-m** [comment]
Ah, good, bookmark then. It looks extremely European to me, and we did not record because it didn't occur yet in manuscripts we described. It is apparently present in all of Bruce's manuscripts. I am not sure whether "sewn to the headband" is the best possible description :) it is visible between headband and textblock (I think completely typical for European books).

**Comment by DenisNosnitsin1970** [comment]
Then say that it is most probably European, and that will be enough, you do not need to describe further small details (you can use the formula "attached to"; or visible - in what sense, it is placed? "between headband and textblock" is difficult to figure out. Modern libraries - here Bodleian- destroyed old bindings, mutilated the textblock and infected the volume with alien objects :) thank you, Bodleianers :) ).

**Comment by thea-m** [comment]
Thank you!

**Comment by thea-m** [comment]
Can this be merged?

**Pull request merged**